components/application-operator/pkg/application-controller/reconciler_test.go

```diff
@@ -274,6 +274,8 @@ func TestApplicationReconciler_Reconcile(t *testing.T) {
 		ApplicationReleaseManager := &helmmocks.ApplicationReleaseManager{}
 		ApplicationReleaseManager.On("CheckReleaseExistence", applicationName).Return(true, nil)
 		ApplicationReleaseManager.On("CheckReleaseStatus", applicationName).Return(releaseStatus, statusDescription, nil)
+		ofType := mock.AnythingOfType("*v1alpha1.Application")
```

Comment:
As we don't need to call AssertCalled this can be defied in place as 

`ApplicationReleaseManager.On("UpgradeApplicationRelease", mock.AnythingOfType("*v1alpha1.Application")) `